### PR TITLE
update bibiliography citing Galaxy, from sfb1425

### PR DESCRIPTION
**Pull request by pavanvidem**

### CODE REVIEW

File: _bibliography/citations-eu.bib [hunk 1]
```diff
@@ -6522,3 +6522,94 @@ @article{zirngibl_triose_2023
...
...1 as channels contributing to observed ion currents. MΦ displayed four patterns for outward and two for inward-rectifier potassium currents. Additionally, MΦ showed surface expression of Cx43, a prerequisite for homo- and/or heterotypic electrotonic coupling. Experimental results fed into development of an original computational model to describe cardiac MΦ electrophysiology. Computer simulations to quantitatively assess plausible effects of MΦ on electrotonically coupled CM showed that MΦ can depolarize resting CM, shorten early and prolong late action potential duration, with effects depending on coupling strength and individual MΦ electrophysiological properties, in particular resting membrane potential and presence/absence of Kir2.1.Our results provide a first electrophysiological characterization of cardiac resident MΦ, and a computational model to quantitatively explore their relevance in the heterocellular heart. Future work will be focussed at distinguishing electrophysiological effects of MΦ–CM coupling on both cell types during steady-state and in patho-physiological remodelling, when immune cells change their phenotype, proliferate, and/or invade from external sources.},
```
Comment (bgruening) [inline _bibliography/citations-eu.bib:6527]: this one is already included.

File: _bibliography/citations-eu.bib [hunk 2]
```diff
@@ -6522,3 +6522,94 @@ @article{zirngibl_triose_2023
...
...ch as the IL-6 cytokine family member oncostatin M (OSM), enhance angiogenesis, whereas others, such as ciliary neurotropic factor (CNTF), reduce it. This study aimed to clarify these conflicting effects. In contrast to the anti-angiogenic cytokine CNTF, the pro-angiogenic cytokine OSM was able to activate intracellular signaling pathways beyond the STAT3 pathway, including the ERK and AKT pathways. These differences translated into transcriptomic and metabolic shifts. siRNA-mediated STAT3 knockdown experiments showed a decrease in VEGF-induced endothelial migration and sprouting, enhancing the pro-angiogenic drive of OSM and switching the CNTF response from anti-angiogenic to pro-angiogenic. These effects correlated with a transcriptomic shift representing enhanced STAT1 and ERK activity following STAT3 knockdown, including a compensatory prolonged phosphorylated STAT1 activity. In conclusion, the angiogenic effect of STAT3 appears to be determined by cytokine-induced STAT3 specificity and simultaneous activity of other intracellular signaling pathways, whereas the STAT3 pathway, predominantly recognized for its pro-angiogenic phenotypes, reveals novel anti-angiogenic potential.},
```
Comment (bgruening) [inline _bibliography/citations-eu.bib:6570]: this one is also already in, pleasee check them

File: _bibliography/citations-eu.bib (hunk 1 above)
Comment (pavanvidem) [reply _bibliography/citations-eu.bib:6527]: ok, I will remove it. Should I also reorder them based on the string?

**Comment by bgruening** [comment]
Thanks. So many new looked suspicious :)

**Pull request merged**